examples/ola-streaming-client.cpp

```diff
@@ -78,11 +84,30 @@ int main(int argc, char *argv[]) {
 
   if (FLAGS_dmx.str().empty()) {
     string input;
+    bool have_universe = false;
+    unsigned int read_universe = FLAGS_universe;
+
     while (!terminate && std::cin >> input) {
-      ola::StringTrim(&input);
-      SendDataFromString(&ola_client, FLAGS_universe, input);
+      if (!have_universe && FLAGS_universe_from_source) {
+        if (!ola::StringToInt(input, &read_universe, true)) {
+          OLA_FATAL << "Could not convert universe number.";
+          exit(1);
+        }
+
+        have_universe = true;
+        continue;
+      }
+
+      if (have_universe || !FLAGS_universe_from_source) {
+        SendDataFromString(&ola_client, read_universe, input);
+        have_universe = false;
+      }
     }
   } else {
+    if (FLAGS_universe_from_source) {
+      OLA_FATAL << "Not reading from STDIN. Use -u to specify universe.";
+      exit(1);
```

Comment:
It would be better to `exit(ola::EXIT_USAGE);`

Reply:
Cool - I'll cleanup the other users of `exit(1)` as well, then.